dev/RadioButtons/RadioButtons.xaml

```diff
@@ -19,6 +19,7 @@
                         <local:ItemsRepeater x:Name="InnerRepeater" ItemTemplate="{TemplateBinding ItemTemplate}">
                             <local:ItemsRepeater.Layout>
                                 <primitives:ColumnMajorUniformToLargestGridLayout
+                                    MaximumColumns="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=MaximumColumns}"
```

Comment:
This is fine but I'm curious why TemplateBinding doesn't work here?

Reply:
I am also curious, I tried and it did not but i didn't spend time debugging why.